### PR TITLE
99namesofallah

### PR DESCRIPTION
**Pull request by shakil2019**

### CODE REVIEW

**Comment by shakil2019** [comment]
#1 working on this task

File: src/js/main/main.js [hunk 1]
```diff
@@ -81,4 +81,8 @@ function getURLParam(param) {
     }
 
     return (false);
-}
\ No newline at end of file
+}
+
+setTimeout(function(){
```
[the remark this answers is not in the document]
Comment (shakil2019) [reply src/js/main/main.js:86]: Yes we can remove this and thanks for notice to me. i have added just for testing 'll remove then revert back to you. 

> First of all, all new features or big changes should be discussed before implementation
Acknowledge

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,11 @@
 
 <body>
     <main>
-        <div class="content rtl" title="" aria-label=""></div>
+        <div class="content-wrapper">
```
[the remark this answers is not in the document]
Comment (shakil2019) [reply src/index.html:14]: noted.

**Comment by junaidkbr** [comment]
@shakil2019 I see you have resolved the issues but I don't see any changes from you in this PR. I can still see the code I requested to revert.

**Comment by junaidkbr** [comment]
Closing this PR as changes requested in this PR are available in #19

**Pull request closed without merging**